src/documents/activities/CauseAndEffect.html.md.handlebars

```diff
@@ -4,35 +4,39 @@ layout: content
 category: Activities
 ---
 
-This activity demonstrates the potential to have a broader beneficial impact with our everyday actions. The challenge is to define a possible chain of events that could take you from a specific cause to an effect.
+This activity demonstrates the potential to have a broader beneficial impact with our everyday actions. The challenge is
```

Comment:
Also additional change.

Reply:
these are the spelling changes

Reply:
Why deleting the 7th line and adding it? I dont think there is any kind of spelling mistake in that.